src/components/d2l-activity-exemptions.js

```diff
@@ -0,0 +1,313 @@
+import '@brightspace-ui/core/components/alert/alert-toast.js';
+import '@brightspace-ui/core/components/button/button.js';
+import '@brightspace-ui/core/components/inputs/input-checkbox.js';
+import '@brightspace-ui/core/components/inputs/input-search.js';
+
+import { css, html, LitElement } from 'lit-element/lit-element';
+import { BaseMixin } from '../mixins/base-mixin';
+import { ExemptionManager } from '../services/exemptionManager';
+import { tableStyles } from '@brightspace-ui/core/components/table/table-wrapper.js';
+
+class ActivityExemptions extends BaseMixin(LitElement) {
+	static get properties() {
+		return {
+			autoload: {
+				type: Boolean,
+				attribute: 'autoload'
```

Comment:
nit: this is redundant since it's the same as the property name

Reply:
Well, it defaults to false, so you probably want 
```suggestion
				attribute: true
```